hil/client/base.py

```diff
@@ -71,13 +72,19 @@ def check_response(self, response):
         except ValueError:
             return response.content
 
-    def find_reserved(self, string):
+    def _find_reserved(self, string, slashes_ok=False):
         """Returns a list of illegal characters in a string"""
-        p = r"[^A-Za-z0-9 $_.+!*'(),-]+"
-        return list(x for l in re.findall(p, string) for x in l)
-
-    def find_reserved_w_slash(self, string):
-        """Returns a list of illegal characters in a string
-        excluding `/` for channels and ports"""
-        p = r"[^A-Za-z0-9 /$_.+!*'(),-]+"
-        return list(x for l in re.findall(p, string) for x in l)
+        if slashes_ok:
+            p = r"[^A-Za-z0-9 /$_.+!*'(),-]+"
+            return list(x for l in re.findall(p, string) for x in l)
+        else:
+            p = r"[^A-Za-z0-9 $_.+!*'(),-]+"
+            return list(x for l in re.findall(p, string) for x in l)
```

Comment:
you could just do a single return after the if-else block

Reply:
Fixed